parsing/pcoq.ml

```diff
@@ -587,12 +587,31 @@ let recover_grammar_command (type a) (tag : a grammar_command) : a list =
 
 let extend_dyn_grammar (GrammarCommand.Dyn (tag, g)) = extend_grammar_command tag g
 
+(* Tactic parsing modes *)
+let add_proof_tactic_entry p e = Hashtbl.add tactic_entry p e
```

Comment:
Actually despite my comment below, I think there is a way to do that without incurring into awful sins, but I need to try.